packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/DepositMethods/BankDeposit/index.tsx

```diff
@@ -59,12 +59,32 @@ const BankDeposit = ({ icon, onClick, value }: Props) => (
             />
           </DisplayTitleBank>
           <DisplaySubTitle>
-            <FormattedMessage id='copy.instantly_available' defaultMessage='Instantly Available' />
+            {value.currency === 'GBP' ? (
```

Comment:
Could you define a constant for each hardcoded string? I see those strings duplicated everywhere

Reply:
This is a string, but it is being compared to a union type, so typescript is checking the type of the "string" here.
should I move to a constant event in that case?

Reply:
A constant will still be string type, so yes, it will be great to improve this